public/scripts.js

```diff
@@ -8,36 +8,164 @@
     })
   }
 
-  const getVouchers = function () {
-    return $.get('/vouchers.json')
+  const ProductModel = function (price, countChangeHandler) {
+    this.price = 0
+    this.count = 1
+
+    const $productsCountInput = $('#products-count')
+
+    this.init = function (price) {
+      const _self = this
+      _self.price = price
+      _self.count = $productsCountInput.val()
+      _self.countChangeHandler = countChangeHandler
+
+      $productsCountInput.on('change', function () {
+        _self.count = $(this).val()
+        _self.countChangeHandler(_self.count, _self.price)
+      })
+
+      return _self
+    }
+
+    this.getPrice = function () {
```

Comment:
Original idea was completely different. It's just an artifact left behind. I Will fix it with next code cleanup